app/models/user_authorization_document.rb

```diff
@@ -0,0 +1,21 @@
+class UserAuthorizationDocument < AuthorizationDocument
+  OPTIONAL_DOCUMENTS = [
+    :certificates,
+    :last_year_activities_report,
+    :organization_current_plan
+  ]
```

Comment:
Because it's only three `optional_documents`, what do you think of doing the following instead?
```Ruby
OPTIONAL_DOCUMENTS = %i(certificates last_year_activities_report organization_current_plan)
```